src/Articles.jsx

```diff
@@ -26,10 +31,12 @@ function Articles({ topics }) {
   }, [filter]);
 
   const formatTime = (timeISO) => {
-    let newString = "";
-    const newDate = timeISO.slice(0, 10);
-    const newTime = timeISO.slice(11, 16);
-    return newDate + " " + newTime;
+    // let newString = "";
+    // const newDate = timeISO.slice(0, 10);
+    // const newTime = timeISO.slice(11, 16);
+    // return newDate + " " + newTime;
+    const newTime = moment(timeISO).format("dddd, MMMM Do YYYY, h:mm a");
+    return newTime;
```

Comment:
Absolutely LOVE that you implemented moment straight away! But remember to remove the old code that you have commented out!